### ISSUE TITLE
No more node_modules/.bin directory in the debian package

### ISSUE THREAD

**Issue by Skywalker13** [heartsucker/node-deb#101]
Hi

I don't know exactly why and when but it seems that with an update a buster, the node_modules/.bin directory is no longer packaged. Any ideas?

I use the auto strategy (default) and node_modules is present

Thanks

**Comment by Skywalker13** [comment]
I think that's is related to #71 that breaks too much things... no more symlinks in `.bin` then no more `.bin` dir... and with the copy strategy the binaries in `.bin/` are juste not usable anymore because it's no longer symlinks..

**Comment by heartsucker** [comment]
This sounds like it might be related to #71, yeah. I'm a little swamped with work things but I'll see if I can get around to doing a copy-with-dereference.

**Comment by heartsucker** [comment]
Ah #71 was already closed, so those should be referenced. Is the error you're experiencing in this ticket the same error as your mentioned in #71?

**Comment by Skywalker13** [comment]
Yes, after more investigate it's the same. I'm looking for forking and fixing the regression ASAP because I must provide a working package today.

**Comment by Skywalker13** [comment]
With this "revert" https://github.com/Xcraft-Inc/node-deb/commit/a317f931d10ff7359d04b51e5965e7c3347c350e it works as expected with the copy strategy. But with the auto strategy, I continue to lose the .bin directory :-(

**Comment by heartsucker** [comment]
Ok so would the solution to fix this for everyone be to:

- do not dereference symlinks in `node_modules/.bin`
- dereference all other symlinks

**Comment by Skywalker13** [comment]
It should be fine most of times, even if IMO dereferencing symlinks looks wrong to me.
